rakelib/user_email_identifier.rake

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'roo'
+
+desc 'Build a CSV of user email identifiers'
+
+namespace :user_email_identifier, type: :task do
+  task :build_csv, %i[uuid_list] => :environment do |_, args|
+    xlsx = Roo::Excelx.new(args[:uuid_list])
+    xlsx.sheet(0)
+    row_count = xlsx.last_row - 1
+    puts "[UserEmailIdentifier] Building CSV from #{args[:uuid_list]}, processing #{row_count} rows..."
+
+    CSV.open('./tmp/user_email_identifiers.csv', 'w') do |csv|
+      csv << %w[data_type form_type va_gov_submission_created_at va_gov_user_uuid va_gov_credential_email]
+      xlsx.each_row_streaming(offset: 1, pad_cells: true) do |row|
+        next if row[3].nil? || row[3].cell_value.nil?
```

Comment:
❓ What's special about `row[3]`? Sorry, I'm not familiar with parsing Excel files.